charts/consul/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: consul
 version: 0.34.1
-appVersion: 1.10.2
+appVersion: 1.11.0-alpha
```

Comment:
Given we need to update these values to 1.11.0-alpha for these tests to run, does it make sense to merge this PR? 🤔

Reply:
I guess i could explicitly set the image in the CI pipelines? that way only the PRs run against 1.11 but nightlies use 1.10